fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -115,14 +114,9 @@ public DataRequestHandler buildWorkflow() {
         }
 
         // If query caching is enabled, the cache is checked before sending the request
-        if (BardFeatureFlag.DRUID_CACHE.isOn()) {
-            if (BardFeatureFlag.DRUID_CACHE_V2.isOn()) {
-                uiHandler = new CacheV2RequestHandler(uiHandler, dataCache, querySigningService, mapper);
-                nonUiHandler = new CacheV2RequestHandler(nonUiHandler, dataCache, querySigningService, mapper);
-            } else {
-                uiHandler = new CacheRequestHandler(uiHandler, dataCache, mapper);
```

Comment:
If we're only deprecating cache v1, rather than removing it, we should leave this conditional here and log a warning if someone uses it, rather than removing it.